config/internal/visualization-server/imagestream-tensorflow.yaml.tmpl

```diff
@@ -0,0 +1,26 @@
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  annotations:
+    opendatahub.io/notebook-image-url: "https://github.com/red-hat-data-services/notebooks/blob/main/jupyter/tensorflow/ubi8-python-3.8"
```

Comment:
Another location with downstream references. Are there no ODH tensorflow images we can use instead?

Reply:
removed visualization server so this is not needed